diff --git a/node_modules/@react-three/fiber/dist/declarations/src/three-types.d.ts b/node_modules/@react-three/fiber/dist/declarations/src/three-types.d.ts
index e0d3b15..c9a68f3 100644
--- a/node_modules/@react-three/fiber/dist/declarations/src/three-types.d.ts
+++ b/node_modules/@react-three/fiber/dist/declarations/src/three-types.d.ts
@@ -38,14 +38,25 @@ export type ThreeToJSXElements<T extends Record<string, any>> = {
 };
 type ThreeExports = typeof THREE;
 type ThreeElementsImpl = ThreeToJSXElements<ThreeExports>;
-export interface ThreeElements extends Omit<ThreeElementsImpl, 'audio' | 'source' | 'line' | 'path'> {
+export interface ThreeElements {
     primitive: Omit<ThreeElement<any>, 'args'> & {
         object: object;
     };
-    threeAudio: ThreeElementsImpl['audio'];
-    threeSource: ThreeElementsImpl['source'];
-    threeLine: ThreeElementsImpl['line'];
-    threePath: ThreeElementsImpl['path'];
+    mesh: any;
+    group: any;
+    scene: any;
+    boxGeometry: any;
+    sphereGeometry: any;
+    planeGeometry: any;
+    cylinderGeometry: any;
+    meshStandardMaterial: any;
+    meshBasicMaterial: any;
+    meshPhysicalMaterial: any;
+    ambientLight: any;
+    directionalLight: any;
+    pointLight: any;
+    spotLight: any;
+    hemisphereLight: any;
 }
 declare module 'react' {
     namespace JSX {
